src/DynamoInstall/Release.xsl

```diff
@@ -47,6 +47,10 @@
     <xsl:key name="moq-search" match="wix:Component[contains(wix:File/@Source, 'Moq')]" use="@Id"/>
     <xsl:template match="wix:Component[key('moq-search', @Id)]" />
 
+    <!--Exclude Caste.Core.dll-->
+    <xsl:key name="castle-search" match="wix:Component[contains(wix:File/@Source, 'Castle.Core')]" use="@Id"/>
+    <xsl:template match="wix:Component[key(castle-search', @Id)]" />
+
```

Comment:
What is this about?

Reply:
oh..the Moq dependency.
I've tried before to solve the issue of test dependencies in the final output path but I could not find a one size fits all solution.
Maybe the easiest way is to put everything in there, then just filter out (but I know some file escaped in the past and we got stuck with them)

Reply:
yeah, I mention a few in the pr description - I don't intend to solve it here, but I don't want to make it worse.

Reply:
I ended up continuing with your approach - `Moq`'s new deps are copied to the test_dependencies folder. In addition `Moq` is also copied to the bin folder since we cannot remove it until a GL release.

I've also added a bunch of `SetupFixtures` that hook up the resolver you added for other test assemblies that started failing now that the new moq deps are not in the bin folder.